bridge/bridge.sh

```diff
@@ -0,0 +1 @@
+python3 -u "./bridge.py"
```

Comment:
running `python3` command on my PC just opens the Python program on the Windows store, I run python with `python` or `py`

Reply:
install-host.sh is for linux and macOS. Windows users should install it using install-host.bat.